src/main/java/com/jcabi/http/request/FakeRequest.java

```diff
@@ -116,8 +118,7 @@ public Response send(final Request req, final String home,
     /**
      * Content received.
      */
-    @Immutable.Array
-    private final transient byte[] content;
+    private final transient Map<Pattern, byte[]> content;
```

Comment:
@dmzaytsev since we construct new `FakeRequest` instances when adding new regexps, can